config/dev-dgraeber.env

```diff
@@ -0,0 +1,2 @@
+export GLOBAL_CUSTOM_DEPLOYMENT_NAME_IDENTIFIER="dgraeber"
```

Comment:
I am thinking that these would be good in the examples dir....rather than local development

Reply:
This a config file to help with the local dev. I would say, it should either be wherever it is or moved under `manifests/local` directory